CI/windows/build_ffmpeg.sh

```diff
@@ -94,6 +113,9 @@ _build_product() {
         --enable-libvorbis \
         --enable-libvpx \
         --enable-libsrt \
+        --enable-librist \
+        --$SWITCH-libaom \
```

Comment:
Is there an issue with libaom on Windows 32 bits or it's a typo ?

Reply:
no issue, but Jim decided not to ship aom with 32 builds